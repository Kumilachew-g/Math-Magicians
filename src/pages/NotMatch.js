import React from 'react';
import image from '../images/notFound.jpg';

const NotMatch = () => (
  <div className="no-match">
    <h3>Whoops! Page not found!</h3>
    <img src={image} alt="Not found" />
  </div>
);

export default NotMatch;
