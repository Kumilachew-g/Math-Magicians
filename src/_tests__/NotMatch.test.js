import React from 'react';
import renderer from 'react-test-renderer';
import NotMatch from '../pages/NotMatch';

it('expect notmatch page to match working snapshot', () => {
  const notMatchPage = renderer.create(<NotMatch />).toJSON();
  expect(notMatchPage).toMatchSnapshot();
});
