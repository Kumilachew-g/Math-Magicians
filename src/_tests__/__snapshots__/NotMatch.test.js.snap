// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect notmatch page to match working snapshot 1`] = `
<div
  className="no-match"
>
  <h3>
    Whoops! Page not found!
  </h3>
  <img
    alt="Not found"
    src="notFound.jpg"
  />
</div>
`;
